store/account_test.go

```diff
@@ -118,7 +118,7 @@ func TestAccountByNumber(t *testing.T) {
 func TestAccountByAddress(t *testing.T) {
 	setup(t)
 
-	total := util.RandInt32(100)
+	total := util.RandInt32(100) + 1
```

Comment:
Opps, We should do it for `TestIterateAccounts` test. No har to have it here. Is there any other places?

Reply:
Yes


**store/validator_test.go**

```go
func TestAccountBatchSaving(t *testing.T) {
	setup(t)

	total := util.RandInt32(100)
	t.Run("Add some accounts", func(t *testing.T) {
		for i := int32(0); i < total; i++ {
			acc, signer := account.GenerateTestAccount(i)
			tStore.UpdateAccount(signer.Address(), acc)
		}
		assert.NoError(t, tStore.WriteBatch())
		assert.Equal(t, tStore.TotalAccounts(), total)
	})

	t.Run("Close and load db", func(t *testing.T) {
		tStore.Close()
		store, _ := NewStore(tStore.config, 21)
		assert.Equal(t, store.TotalAccounts(), total)
	})
}
```

Reply:
> Opps, We should do it for `TestIterateAccounts` test. No harm to have it here. Is there any other places?

If `total := 0` for-loop has been skip.